blazor/diagram/connectors/segments/multiplesegments.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: Multiple Segments in Blazor Diagram Component | Syncfusion
+description: Checkout and learn here all about Multiple Segments in Syncfusion Blazor Diagram component and much more details.
```

Comment:
description: Check out and learn about Multiple Segments in the Syncfusion Blazor Diagram component and much more details.